doc/animation_widget_article/article.md

```diff
@@ -118,18 +118,18 @@ Let's you have your `build` method in one of your pages; pretty normal Flutter s
   }
 ```
 
-Note that it could be any component, however complex, inside your widgets tree. Note also that I have omitted the "magic" of the equation here. How is it that we create a component for an animation? Very basically (more details in the flame tutorial), Flame provides components, one of which is the `SpriteAnimationComponent` that receives a `SpriteAnimation` object describing the animation and does exactly what we want. All components live inside a `Game` instance, that can add custom logic relating to the game loop. For our case, we just want to create a simple, empty game and add a single `AnimationComponent` with a simple `Animation` inside. So Flame provides a helper to do that, the `Flame.util.animationAsWidget` method. It takes the size of the object as a Flame's `Position` instance (a generic class to represent a pair of doubles), and also takes in an `Animation` instance representing our frame list. To use that, let's import both `Flame` and the `Animation` class. However, since Flutter adds it's own animation classes, let's use an alias in order to not mess up the names. Therefore, add these imports to the top of the file:
+Note that it could be any component, however complex, inside your widgets tree. Note also that I have omitted the "magic" of the equation here. How do we create a component for an animation? Very basically (more details in the flame tutorial), Flame provides components, one of which is the `SpriteAnimationComponent` that receives a `SpriteAnimation` object describing the animation and does exactly what we want. All components live inside a `Game` instance, that can add custom logic relating to the game loop. For our case, we just want to create a simple, empty game and add a single `AnimationComponent` with a simple `Animation` inside. So Flame provides a helper to do that, the `Flame.util.animationAsWidget` method. It takes the size of the object as a Flame's `Position` instance (a generic class to represent a pair of doubles), and also takes in an `Animation` instance representing our frame list. To use that, let's import both `Flame` and the `Animation` class. However, since Flutter adds it's own animation classes, let's use an alias in order to not mess up the names. Therefore, add these imports to the top of the file:
 
 ```dart
 import 'package:flame/sprite_animation.dart'; // imports the SpriteAnimation class
 import 'package:flame/flame.dart'; // imports the Flame helper class
-import 'package:flame/position.dart'; // imports the Position class
+import 'package:flame/vector2.dart'; // imports the Vector2 class
```

Comment:
are we doing package:vector_math/vector_math_64.dart or package:flame/vector2.dart? I'm confused now

Reply:
for reals we can just get rid of this article, it's not even updated any more, Erick changed that API

Reply:
`package:vector_math/vector_math_64.dart`, the `vector2.dart` class exports that class and the helper functions.
Maybe it is nicer not to export the class from there?